/*
 * Contains all of our custom exceptions
 */
package de.exxcellent.challenge.elaborate.exceptions;
